packages/webview_flutter/test/webview_flutter_test.dart

```diff
@@ -254,19 +254,49 @@ void main() {
     final FakePlatformWebView platformWebView =
         fakePlatformViewsController.lastCreatedView;
 
-    expect(platformWebView.currentUrl, 'https://youtube.com');
+    expect(platformWebView.url, 'https://youtube.com');
 
     controller.loadUrl('https://flutter.io');
 
-    expect(platformWebView.currentUrl, 'https://flutter.io');
+    expect(platformWebView.url, 'https://flutter.io');
 
     controller.goBack();
 
-    expect(platformWebView.currentUrl, 'https://youtube.com');
+    expect(platformWebView.url, 'https://youtube.com');
 
     controller.goForward();
 
-    expect(platformWebView.currentUrl, 'https://flutter.io');
+    expect(platformWebView.url, 'https://flutter.io');
+  });
+
+
+  testWidgets('Current URL', (WidgetTester tester) async {
+    WebViewController controller;
+    await tester.pumpWidget(
+      WebView(
+        onWebViewCreated: (WebViewController webViewController) {
+          controller = webViewController;
+        },
+      ),
+    );
+
+    expect(controller, isNotNull);
+
+    // Test a WebView without an explicitly set first URL.
+    expect(await controller.currentUrl(), null);
+
+    final FakePlatformWebView platformWebView =
```

Comment:
Please note: I was a little uncertain how these tests were actually done with FakePlatformWebView. Is it possible that by me adding the currentUrl accessor, that FakePlatformWebView is no longer needed? If so this part of the code will need to be changed.

Reply:
Yes it's actually nice that we can use the public API for testing here.
We still need the `FakePlatformWebView` implementation so something responds to the method channel.
But yes, in this test (and the other tests that only used the `FakePlatformWebView` directly to get the current url) we don't need to fetch `fakePlatformViewsController.lastCreatedView` any more.

Sweet 😄

Reply:
Actually if we're migrating the other tests that used `platformWebView.currentUrl` to instead do `controller.currentUrl()` we probably don't need a 'Current URL' test.

Reply:
updated!